force-app/main/default/classes/FeatureParameters.cls

```diff
@@ -1,8 +1,16 @@
+/* When a new telemetry feature parameter needs to be added we need to modify this class and do the following : 
+1. Need to add a new DeveloperName enum.
+2. Need to add a new when statement to the makeFeatureParameter method.
+3. Need to create a new inner class similar to the other inner classes added for feature parameters and 
+update the methods within the inner class accordingly
+4. Modify FeatureParameters_TEST class to add tests for the newly created inner class*/
+
```

Comment:
@bmuniswamy  - should we also add the step about manually creating the featureParameter metadata?

Reply:
Yeah that is a good callout @jjbennett. I will add that